channels/CMakeLists.txt

```diff
@@ -155,10 +155,6 @@ macro(add_channel_client_subsystem _channel_prefix _channel_name _subsystem _typ
 endmacro(add_channel_client_subsystem)
 
 macro(add_channel_client_library _module_prefix _module_name _channel_name _dynamic _entry)
-	if(${_dynamic} AND MSVC AND (NOT STATIC_CHANNELS))
```

Comment:
@bmiklautz Did prevent the build to succeed (the file is not generated / available), symbols of channels are exported using ```__declspec(dllexport)``` anyway.